src/single-spa-vue.js

```diff
@@ -47,6 +51,10 @@ function mount(opts, mountedInstances, props) {
         otherOptions.el = props.domElement;
       }
 
+      if (!opts.appOptions.render && !opts.appOptions.template && opts.rootComponent) {
+        otherOptions.render = (h) => h(opts.rootComponent)
```

Comment:
Oh I didn't even realize this was an option for rendering Vue apps. I sorta like this way better than the template way.

Let's update the documentation to encourage people down this route instead, with loadRootComponent or rootComponent?

Reply:
Also I think we should change the documentation from exporting arrays for the lifecycles to just exporting the one function -- no need to make it more complicated than it has to be, i think

Reply:
@joeldenning if you want to see some good examples of this dig into the vuetify git repository.  great work on this lads, you saved me so much heart ache :D